### DISCUSSION TITLE
Item padding (line height) in Clipboard History

### DISCUSSION THREAD

**Discussion by MeX000** [hluk/CopyQ#2036] [Q&A]
Hello.
Can anyone point to correct parameters (syntax) for making lines more compact?
![1111](https://user-images.githubusercontent.com/12931005/177750278-1ffeabe5-28eb-4382-9304-69d3f9647dd0.png)

**Comment by hluk** [comment]
Try clicking on the Edit Theme button and setting `item_spacing=0` in the editor.

**Reply by MeX000** [reply]
Thanks, already tried it.
In the end I just checked History → Show simple items.
Now the view is compact as I wanted.
